src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const styles = sourceString
+    .split(';')
```

Comment:
The split operation and subsequent filter can be chained with proper indentation for better readability. Also, ensure that you handle the case where a style declaration might not have a value or is malformed.